app/keepers/keepers.go

```diff
@@ -532,6 +537,16 @@ func (ak *AppKeepers) InitKeepers(
 		ak.IBCKeeper.PortKeeper, ak.AccountKeeper, ak.BankKeeper,
 	)
 
+	ak.RatelimitKeeper = *ratelimitkeeper.NewKeeper(
```

Comment:
lets remember to pass `ak.RatelimitKeeper` to `ak.IBCFeeKeeper` as ics4wrapper

Reply:
no longer needed cause IBCFeeKeeper was removed